.github/workflows/tf-aws-test.yml

```diff
@@ -179,12 +180,28 @@ jobs:
             elif ${{ env.LOCAL_BUILD == 'false'}}
             then 
               WING_CLI=$(which wing)
+              # COMPETIBILITY="-t @winglang/compatibility-spy" //TODO: will be handled in a following PR
```

Comment:
`@winglang/compatibility-spy` is currently published, does this already work?

Reply:
yes, it works, but I want to test it separately